src/modules/dashboard/components/ColonyHome/ColonyMembers/MembersSubsection.tsx

```diff
@@ -127,30 +128,38 @@ const MembersSubsection = ({
 
   const setHeading = useCallback(
     (hasCounter) => (
-      <Tooltip
-        content={
-          <div className={styles.tooltip}>
-            <FormattedMessage
-              {...MSG.tooltipText}
-              values={{ isContributors }}
+      <div className={styles.heading}>
+        <Tooltip
+          content={
+            <div className={styles.tooltip}>
+              <FormattedMessage
+                {...MSG.tooltipText}
+                values={{ isContributors }}
+              />
+            </div>
+          }
+        >
+          <NavLink to={membersPageRoute}>
+            <Heading
+              appearance={{ size: 'normal', weight: 'bold' }}
+              text={MSG.title}
+              textValues={{
+                count: members?.length,
+                hasCounter,
+                isContributors,
+              }}
             />
-          </div>
-        }
-      >
-        <NavLink to={membersPageRoute}>
-          <Heading
-            appearance={{ size: 'normal', weight: 'bold' }}
-            text={MSG.title}
-            textValues={{
-              count: members?.length,
-              hasCounter,
-              isContributors,
-            }}
+          </NavLink>
+        </Tooltip>
+        {!isContributors && (
```

Comment:
You are correct, it's available for everyone! `isContributors` is a bool that tells us if we are rendering the contributors heading: 
![FireShot Capture 598 - Actions - Colony - wonker - localhost](https://user-images.githubusercontent.com/18473896/169359626-65d0d685-16be-4715-b774-9f6d484b4428.png)

If we are not, that means that we are rendering the watchers heading:
![FireShot Capture 599 - Actions - Colony - wonker - localhost](https://user-images.githubusercontent.com/18473896/169359781-7d0941e4-f248-41d9-b68d-0c451f191d4a.png)

With your comment, and going by the name of `isContributors`, I was also confused because I didn't remember restricting the link, so I think it's better if we change the name of this variable to avoid further confusion whenever someone works on this part of the code.